src/alarm/alarm_instance.cc

```diff
@@ -165,14 +165,16 @@ void AlarmInstance::HandleGetAlarm(const picojson::value& msg) {
 
 void AlarmInstance::HandleGetAllAlarms() {
   std::vector<std::shared_ptr<AlarmInfo> > alarms;
+  picojson::array alarm_infos;
```

Comment:
Moving this declaration isn't really needed. You can also remove the empty line between `alarms` and `int ret = ...` for consistency. After that, lgtm.